open-build-service-mock/src/api/mod.rs

```diff
@@ -92,6 +97,13 @@ fn check_auth(auth: &BasicAuth, request: &Request) -> Result<(), ApiError> {
     }
 }
 
+fn find_query_param<'r, 'n>(request: &'r Request, name: &'n str) -> Option<Cow<'r, str>> {
```

Comment:
```suggestion
fn find_query_param<'r>(request: &'r Request, name: &str) -> Option<Cow<'r, str>> {
```

you should be able to ellide the 'n lifetime here as it doesn't need tying up/constraining

Reply:
oh nice, I didn't realize you could elide only some lifetimes vs all of them! (Slightly surprised clippy didn't yell at me here though)